project/plugins.sbt

```diff
@@ -1,9 +1,8 @@
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"            % "2.8.21")
+addSbtPlugin("org.playframework" % "sbt-plugin"            % "3.0.2")
 addSbtPlugin("org.scalameta"     % "sbt-scalafmt"          % "2.0.0")
-addSbtPlugin("com.typesafe.sbt"  % "sbt-twirl"             % "1.5.1")
-addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.9")
+
+addSbtPlugin("com.github.sbt" % "sbt-native-packager" % "1.9.16")
+addDependencyTreePlugin
 
 //this is required because scala compiler depends on a higher version of scala-xml than sbt-native-packager dependencies
```

Comment:
Maybe delete the comment as well as the code?

Reply:
Thanks for spotting 👀